pkg/controller/build/buildrequest/buildrequest.go

```diff
@@ -198,6 +218,92 @@ func (br buildRequestImpl) additionaltrustbundleToConfigMap() *corev1.ConfigMap
 	return configmap
 }
 
+func (br buildRequestImpl) etcPolicyToConfigMap(mc *mcfgv1.MachineConfig) (*corev1.ConfigMap, error) {
+	// Build the ConfigMap data
+	configMapData, err := br.getConfigMapData(mc, "/etc/containers/policy.json", "/etc/containers/")
+	if err != nil {
+		return nil, err
+	}
+	if len(configMapData) == 0 {
+		return nil, nil
+	}
+
+	// Create the ConfigMap
+	configmap := &corev1.ConfigMap{
+		TypeMeta:   metav1.TypeMeta{},
+		ObjectMeta: br.getObjectMeta(br.getEtcPolicyConfigMapName()),
+		Data:       configMapData,
+	}
+	return configmap, nil
+}
+
+func (br buildRequestImpl) etcRegistriesToConfigMap(mc *mcfgv1.MachineConfig) (*corev1.ConfigMap, error) {
+	// Build the ConfigMap data
+	configMapData, err := br.getConfigMapData(mc, "/etc/containers/registries.conf", "/etc/containers/")
+	if err != nil {
+		return nil, err
+	}
+	if len(configMapData) == 0 {
+		return nil, nil
+	}
+
+	// Create the ConfigMap
+	configmap := &corev1.ConfigMap{
+		TypeMeta:   metav1.TypeMeta{},
+		ObjectMeta: br.getObjectMeta(br.getEtcRegistriesConfigMapName()),
+		Data:       configMapData,
+	}
+	return configmap, nil
+}
+
+func (br buildRequestImpl) getConfigMapData(mc *mcfgv1.MachineConfig, filePath, prefixToTrim string) (map[string]string, error) {
```

Comment:
thought (non-blocking): Perhaps we should name this something like, `ignitionFileToConfigMapData()` since this name implies that it's used in a variety of different situations?

Reply:
Fixed